app/views/user_mailer/password_reset.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Reset Password</h1>
+<p>To reset your password, click the URL below.</p>
+
+<p><%= edit_password_reset_url(@user.password_reset_token, :host => "localhost:3000") %></p>
```

Comment:
It's best not to hard code the host in the view file. Please put it in the `config/environments/development.rb`